test/common/tests.go

```diff
@@ -106,7 +106,7 @@ var (
 
 		{
 			[]TestCase{
-				{"A34 - Verify SKU values", TestQuotaValues},
+				{"A34 - Verify QUOTA values", TestQuotaValues},
```

Comment:
This looks ok but it is is dependent on https://github.com/integr8ly/integreatly-operator/pull/1688

Reply:
@trepel that on should be passing now as the branch i'm based this pr on is the branch for the pr you've linked above.